README.md

```diff
@@ -60,6 +65,22 @@ In this case you can define a regular expression, which matches all user agent s
 ```
 This regular expression would match any user agent string, that ends with *My Example Webapp*.
 
+### Add a separate links specific to some device/platform
+
+There are cases when there are separate applications meant for phone and tablet of the same OS. For this purpose you can use the OS related metatag with platform-device suffix, e.g.
+
+#### iOS - iPhone app link can be set with the following metatag:
+```html
+  <meta name="smartbanner:button-url-apple-iphone" content="https://itunes.apple.com/iphonelink">
+```
+
+#### iOS - iPad app link can be set with the following metatag:
+```html
+    <meta name="smartbanner:button-url-apple-ipad" content="https://itunes.apple.com/ipadlink">
+```
+
+**Note:** Is is important that if 'global' links, like `<meta name="smartbanner:button-url-apple"/>` or `<meta name="smartbanner:button-url-google"/>` will be set, device related app links will be omitted (-phone; -tablet).
```

Comment:
Strange, I tested it on my Android and it works fine - while the logic for iPhone is quite similar, except the userAgent of course. Unfortunately I don't have an iPhone nor iPad, so I don't have a chance to test it against real devices, but in Chrome Developer tools against different devices, including iPhone and iPad it works fine as well.

Also the tests are passing against specific userAgent, so not sure why in your case it was not working.

Anyway, I will take a look on option to include a Global link as fallback.